### PR TITLE
Add azure fetcher to testing environment

### PR DESCRIPTION
**Pull request by copyhold**

### CODE REVIEW

File: testing-configs/fetcher-azure.json [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+  "type": "FETCHER",
+  "subtypes": [
+    {
+      "name": "Azure",
+      "logo": "https://dytvr9ot2sszz.cloudfront.net/logz-docs/shipper-logos/google-cloud-monitoring.png",
+      "supportedOs": [
+        "Fetcher"
+      ],
+      "integrationTemplate": "https://ssh.cloud.google.com/cloudshell/editor?cloudshell_git_repo=https://github.com/logzio/logzio-google-metrics",
+      "required": false,
```
Comment (ifatregev) [inline testing-configs/fetcher-azure.json:13]: what is this?

File: testing-configs/fetcher-azure.json [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+              "variant": "info",
+              "title": "Connect your Microsoft Azure app to Logz.io",
+              "text": "To get started with Microsoft Azure and Logz.io, you’ll need to create an app and connect it. See our step-by-step guide."
+            }
+          ],
+          "params": [
+            {
+              "name": "tenant.id",
+              "hint": null,
+              "logo": null,
```
Comment (ifatregev) [inline testing-configs/fetcher-azure.json:39]: not needed

File: testing-configs/fetcher-azure.json [hunk 3]
```diff
@@ -0,0 +1,128 @@
...
+      "integrationTemplate": "https://ssh.cloud.google.com/cloudshell/editor?cloudshell_git_repo=https://github.com/logzio/logzio-google-metrics",
+      "required": false,
+      "datasources": [
+        {
+          "name": "Azure Metrics",
+          "hint": null,
+          "logo": "https://dytvr9ot2sszz.cloudfront.net/logz-docs/shipper-logos/google-cloud-monitoring.png",
+          "label": "Azure Metrics",
+          "description": "fetcher description",
+          "required": false,
```
Comment (ifatregev) [inline testing-configs/fetcher-azure.json:21]: ?

File: testing-configs/fetcher-azure.json [hunk 4]
```diff
@@ -0,0 +1,137 @@
...
+              "label": "Client Secret",
+              "required": true,
+              "description": "",
+              "hint": "",
+              "value": ""
+            }
+          ],
+          "telemetries": [
+            {
+              "type": "LOG_ANALYTICS",
```
Comment (copyhold) [inline testing-configs/fetcher-azure.json:63]: i think it should be log_analy... because the `Trial fetcher` is of this type.

**Review by ifatregev — APPROVED** [review]
Great

**Pull request merged**